doc/xml/daps_user_output.xml

```diff
@@ -277,4 +277,34 @@
    </para>
   </important>
  </sect1>
-</chapter>
+ <sect1 id="sec.daps.user.output.timestamp">
+ <title>Putting the actual date and time into the target document</title>
+ <para>
+ In order to get information about the build date and time, you can use the dbtimestamp tag.
+ The dbtimestamp tag is a so called processing instruction, so it will be processed by Daps, when the document
```

Comment:
* Use `sgmltag` around `dbtimestamp` like this:

 ```xml
<sgmltag class="pi">dbtimestamp</sgmltag>
```

* I would try to avoid the word "tag" here as it is misleading. Maybe start the sentence like this (you need to rephrase the other part):

```
The processing instruction <sgmltag class="pi">dbtimestamp</sgmltag> is processed by &daps; to ...
```